components/terrain/world.cpp

```diff
@@ -75,4 +75,10 @@ void World::updateTextureFiltering()
     mTextureManager->updateTextureFiltering();
 }
 
+void World::clearAssociatedCaches()
+{
+    mTextureManager->clearCache();
```

Comment:
I don't think we need to clear mTextureManager when Land/LandTexture changes. It caches plain Textures, not LandTextures.

Reply:
You're correct.